Demo/MozillaSocial-iOS/MoSoContent.xcodeproj/project.pbxproj

```diff
@@ -498,7 +502,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"MoSoContent/Preview Content\"";
-				DEVELOPMENT_TEAM = EX3VH4YFCH;
+				DEVELOPMENT_TEAM = 4BY3F65HD4;
```

Comment:
we want to keep the team unchanged for now, for distribution purposes

Reply:
oops not sure how that got changed, thanks for the catch!

Reply:
thanks for helping me fix this issue! I reset it back to Read it Later.